bitnami/rabbitmq/README.md

```diff
@@ -111,6 +111,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `auth.tls.overrideCaCertificate`      | Existing secret with certificate content be mounted instead of the `ca.crt` coming from caCertificate or existingSecret/existingSecretFullChain.                        | `""`                                              |
 | `logs`                                | Path of the RabbitMQ server's Erlang log file. Value for the `RABBITMQ_LOGS` environment variable                                                                       | `-`                                               |
 | `ulimitNofiles`                       | RabbitMQ Max File Descriptors                                                                                                                                           | `65536`                                           |
+| `feature_flags`                       | RabbitMQ controls what features are considered to be enabled or available on all cluster nodes. | `drop_unroutable_metric,empty_basic_get_metric,implicit_default_bindings,maintenance_mode_status,quorum_queue,virtual_host_metadata` 
```

Comment:
In the README, the default value is empty. Could you check this?

Reply:
it is there